spec/support/contexts/api/v3/brazil/brazil_flows.rb

```diff
@@ -13,7 +13,8 @@
         api_v3_port1_node,
         api_v3_exporter1_node,
         api_v3_importer1_node,
-        api_v3_country_of_destination1_node
+        api_v3_country_of_destination1_node,
+        api_v3_other_country_of_destination_node
```

Comment:
We cannot do that - there can be only one country of destination in a single flow. In case of Brazil soy, each flow needs to have a path made of 8 nodes and the last of those is the country of destination. What you need here is to create another flow for your test; you could try adding it in the shared context, but it might break other specs that load it, so if that happens add it in your spec file, e.g. 

```
  let!(:api_v3_flow3) do
    FactoryBot.create(
      :api_v3_flow,
      context: api_v3_context,
      path: [
        api_v3_biome_node,
        api_v3_state_node,
        api_v3_municipality_node,
        api_v3_logistics_hub_node,
        api_v3_port1_node,
        api_v3_exporter1_node,
        api_v3_importer1_node,
        api_v3_other_country_of_destination_node
      ].map(&:id),
      year: 2015
    )
  end
```

You need to do a similar thing in order to be able to test filtering by 2 exporters - you need to create an extra flow with a different exporter node, e.g.:

```
  let!(:api_v3_flow4) do
    FactoryBot.create(
      :api_v3_flow,
      context: api_v3_context,
      path: [
        api_v3_biome_node,
        api_v3_state_node,
        api_v3_municipality_node,
        api_v3_logistics_hub_node,
        api_v3_port1_node,
        api_v3_other_exporter_node,
        api_v3_importer1_node,
        api_v3_country_of_destination1_node
      ].map(&:id),
      year: 2015
    )
  end
```